docs/specs/supporting-tools.md

```diff
@@ -0,0 +1,84 @@
+# Supporting tools and services
+
+This page outlines tools and services that currently support the development container specification, including the `devcontainer.json` format. A `devcontainer.json` file in your project tells tools and services that support the dev container spec how to access (or create) a dev container with a well-defined tool and runtime stack. 
+
+While most [dev container properties](devcontainerjson-reference.md) apply to any supporting tool or service, a few are specific to certain tools, which are outlined below.
+
+## Editors
+
+### Visual Studio Code
+
+Visual studio code specific properties go under `vscode` inside `customizations`.
+
+| Property | Type | Description |
+|----------|------|-------------|
+| `extensions` | array | An array of extension IDs that specify the extensions that should be installed inside the container when it is created. Defaults to `[]`. |
+| `settings` | object | Adds default `settings.json` values into a container/machine specific settings file. Defaults to `{}`. |
+
+```
```

Comment:
I was thinking this could be underneath the intro sentence, before the table of properties? To show an example right after we introduce the concept.

Reply:
Moved.